src/Cache/Engine/RedisEngine.php

```diff
@@ -351,6 +359,12 @@ public function clear(): bool
      */
     public function clearBlocking(): bool
     {
+        if ($this->getConfig('flushDb')) {
+            $this->_Redis->flushDB(true);
+
+            return true;
+        }
```

Comment:
Couldn't we expose a new method on `RedisEngine` to flushdb? That would enable `Cache::engine('redis-stuff')->flushDb()` to work.